src/OpenTelemetry.Exporter.Jaeger/JaegerTraceExporter.cs

```diff
@@ -34,14 +34,10 @@ public class JaegerTraceExporter : SpanExporter, IDisposable
         private readonly IJaegerUdpBatcher jaegerAgentUdpBatcher;
         private bool disposedValue = false; // To detect redundant dispose calls
 
-        public JaegerTraceExporter(JaegerExporterOptions options)
+        public JaegerTraceExporter(JaegerExporterOptions options, IJaegerUdpBatcher jaegerAgentUdpBatcher)
```

Comment:
Alternatively we can keep the existing constructor and default to  use `JaegerUdpBatcher`

Reply:
Can you initialize batcher in this constructor and add another constructor that accepts batcher?